hack/ova-provider-server/Containerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.access.redhat.com/ubi9/go-toolset:1.19.10-4 as builder
+FROM registry.access.redhat.com/ubi9/go-toolset:1.19.10-14.1695131433 as builder
```

Comment:
it's not great that the version of go we use here is not aligned with that of bazel - how about converting this to bazel?